mmv1/third_party/terraform/services/vertexai/resource_vertex_ai_feature_online_store_test.go

```diff
@@ -6,78 +6,83 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 
 	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
 )
 
-func TestAccVertexAIFeatureOnlineStore_updated(t *testing.T) {
+func TestAccVertexAIFeatureOnlineStore_vertexAiFeatureonlinestoreWithBigtable_updated(t *testing.T) {
 	t.Parallel()
 
 	context := map[string]interface{}{
-		"random_suffix": acctest.RandString(t, 10),
+		"org_id":          envvar.GetTestOrgFromEnv(t),
```

Comment:
Similar to the example comments, I don't see `org_id` or `billing_account` being used. Do you need them?

Reply:
Updated!